app/views/hyrax/file_sets/_actions.html.erb

```diff
@@ -1,4 +1,4 @@
-<% if can?(:edit, file_set.id) || can?(:destroy, file_set.id) || (Flipflop.enabled?(:annotation) && can?(:download, file_set.id)) %>
+<% if can?(:edit, file_set.id) || can?(:destroy, file_set.id) || Flipflop.enabled?(:annotation) && file_set.pdf? && can?(:download, file_set.id) %>
```

Comment:
Just a little thing, but could you add brackets around the and conditions? Just to make it clear what is connected to what?

```
can?(:edit, file_set.id) || can?(:destroy, file_set.id) || (Flipflop.enabled?(:annotation) && file_set.pdf? && can?(:download, file_set.id))
```

Reply:
I just noticed that you removed the brackets, but that does mean from reading the line i'm not sure exactly what conditions are connected.

Reply:
That was just a mistake on my part during refactoring.  I'll put the brackets back.

Reply:
Addressed below.